guides/common/modules/con_disaster-recovery-with-active-and-passive-project-server-and-external-storage.adoc

```diff
@@ -0,0 +1,6 @@
+[id="disaster-recovery-with-active-and-passive-{project-context}-server-and-external-storage"]
+= Disaster recovery with active and passive {ProjectServer} and external storage
+
+In this scenario, you maintain two {ProjectServer}s and you store critical data externally on a network-attached storage (NAS).
+The primary server is active while the secondary server remains passive.
+If the primary server fails, you attach the NAS to your secondary server and turn it into your new primary server.
```

Comment:
@evgeni They could attach it to both, and then flip it to write for the passive server when it needs to become the primary? Or is this a better /safer method?

Reply:
Depends on how adventurous the user is.
Both attaching and flipping ro/rw is an action you need to perform and as such doesn't give you any real benefit one way or another.
If on the other hand you can keep the storage attached in rw mode all the time, but ensure that absolutely no way the app starts, you win a bit of time/process steps.

Is it worth it? idk.